zend_abstract_interface/zai_string/string.h

```diff
@@ -1,35 +1,50 @@
 #ifndef ZAI_STRING_H
 #define ZAI_STRING_H
 
+// Include this before standard headers, or you may get strange errors like:
+// error: unknown type name 'siginfo_t'
+#include <Zend/zend.h>
+
+#include <zai_assert/zai_assert.h>
```

Comment:
Do we have debug builds running in our CI checking these asserts ?

Reply:
Yes.